scib/integration.py

```diff
@@ -10,6 +10,29 @@
 from .exceptions import OptionalDependencyNotInstalled
 
 
+def harmony(adata, batch, hvg=None, **kwargs):
+    """Harmony wrapper function
+
+    Based on `harmony-pytorch <https://github.com/lilab-bcb/harmony-pytorch>`_ version 0.1.7
+
+    :param adata: preprocessed ``anndata`` object
+    :param batch: batch key in ``adata.obs``
+    :param hvg: list of highly variables to subset to. If ``None``, the full dataset will be used
```

Comment:
Parameter not used, but this seems to be a general for other methods as well, so I'd address this in a separate PR